shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.h

```diff
@@ -133,6 +133,8 @@ FLUTTER_DARWIN_EXPORT
 @property(nonatomic, strong) UITextRange* markedTextRange;
 @property(nonatomic, copy) NSDictionary* markedTextStyle;
 @property(nonatomic, weak) id<UITextInputDelegate> inputDelegate;
+@property(nonatomic, strong) NSMutableArray* pendingDeltas;
+@property(nonatomic, readwrite, copy) NSString* customRunLoopMode;
```

Comment:
It looks like you don't need to expose the run loop mode? IIRC in the macOS implementation it was only exposed because there was a leak in the tests so we couldn't use async expectations.

Reply:
Now that I think about it, do you need to use the runloop API? `dispatch_async` does the same thing right?